Content/index.md

```diff
@@ -60,6 +60,7 @@ Running and managing your own servers can be complex! Cloud providers offer mana
 - [AWS Lambda Events](https://github.com/swift-server/swift-aws-lambda-events)
 - [Deploy to Vercel](https://github.com/swift-cloud/Vercel)
 - [Deploy to Fastly Compute@Edge](https://github.com/swift-cloud/Compute)
+- [Breeze](https://github.com/swift-sprinter/Breeze.git)
```

Comment:
```suggestion
- [Breeze](https://github.com/swift-sprinter/Breeze)
```

link to repo rather than .git link would be better :) 

You may also want to start a thread on https://forums.swift.org/c/server/43 to reach a wide audience :)